apps/system/js/mute_icon.js

```diff
@@ -0,0 +1,25 @@
+/* global BaseIcon */
+'use strict';
+
+(function(exports) {
+  var MuteIcon = function(manager) {
+    BaseIcon.call(this, manager);
+  };
+  MuteIcon.prototype = Object.create(BaseIcon.prototype);
+  MuteIcon.prototype.name = 'MuteIcon';
+  MuteIcon.prototype.update = function() {
+    if (!this.element || !this.enabled()) {
+      return;
+    }
+    var muteState = this.manager.vibrationEnabled ? 'vibration' : 'mute';
+    if (muteState === 'vibration') {
+      this.element.classList.add('vibration');
+    } else {
+      this.element.classList.remove('vibration');
```

Comment:
nit: `this.element.classList.toggle('vibration', muteState === 'vibration');` is easy to read IMHO.